server/controllers/UserEventsActivitiesController.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+//MONGODB
+const { getConnection } = require("../connection/connection");
+
+const dbName = "ParkGames";
+const collectionEvents = "Events";
+const collectionUserEvents = "UserEvents";
+var ObjectId = require("mongodb").ObjectID;
+
+const {
+  getUserFromUserEvents,
+  allEventsArray,
+  getAllEventsUserRegisteredFor,
+  filterEventData,
+} = require("../services/UserEventsActivitiesService");
+
+//@endpoint GET /userActivities
+//@desc get all events that the user joined
+//@access PRIVATE - will need to validate token? YES
+const handleUserActivities = async (req, res, next) => {
+  try {
+    let userId = req.params.userId;
+    if (!userId) {
+      return;
```

Comment:
This is good, but we also need the server to respond here or else it won't finish the request. You can either do 
`res.status(400).send('missing userid')` or `throw new Error('Missing userId')`